enginetest/server_engine.go

```diff
@@ -640,3 +654,12 @@ func enableUserAccounts(ctx *sql.Context, engine *sqle.Engine) error {
 
 	return nil
 }
+
+var cannotBePrepared = map[string]bool{
```

Comment:
These among other queries that errors because they go through `injectBindVarsAndPrepare` causing the non-string sql values to become string values, which causes them to fail. Other queries simply cause incorrect type result, which is not checked for `ServerEngine` test for now.

Reply:
I think this is okay for now, but this definitely seems worthy of a good code comment explaining why we do this and what the next steps are to remove this.